dataservices/service/api/standard.go

```diff
@@ -32,11 +33,15 @@ type Standard struct {
 	metricServicesClient    metricservicesClient.Client
 	userServicesClient      userservicesClient.Client
 	dataFactory             data.Factory
-	dataStore               store.Store
 	dataDeduplicatorFactory deduplicator.Factory
+	dataStore               dataStore.Store
+	taskStore               taskStore.Store
 }
 
-func NewStandard(versionReporter version.Reporter, environmentReporter environment.Reporter, logger log.Logger, metricServicesClient metricservicesClient.Client, userServicesClient userservicesClient.Client, dataFactory data.Factory, dataStore store.Store, dataDeduplicatorFactory deduplicator.Factory) (*Standard, error) {
+func NewStandard(versionReporter version.Reporter, environmentReporter environment.Reporter, logger log.Logger,
```

Comment:
I tried that out, but it added a lot of lines to the code. Plus, if we went to one parameter per line, does that mean we need to change all of the code to reflect that.  I figured it made more sense to just start wrapping the line at some reasonable length. That way, if a function had only 2 or 3 parameters it could remain on the same line.